index.html

```diff
@@ -0,0 +1,224 @@
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24"
+                    class="inline-block w-8 h-8 stroke-current">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                        d="M5 8h14M5 8a2 2 0 110-4h14a2 2 0 110 4M5 8v10a2 2 0 002 2h10a2 2 0 002-2V8m-9 4h4"></path>
+                </svg>
+            </div>
+            <div class="stat-title">New Registers</div>
+            <div class="stat-value">1,200</div>
+            <div class="stat-desc">↘︎ 90 (14%)</div>
+        </div>
+    </div>
+
+    <!-- Call to Action -->
+    <div class="hero py-16 bg-base-200">
+        <div class="hero-content text-center">
+            <div class="max-w-md">
+                <h2 class="text-3xl font-bold">Ready to get started?</h2>
+                <p class="py-6">Join thousands of job seekers who have found their dream jobs through our platform.</p>
+                <div class="space-x-4">
+                    <button class="btn btn-primary btn-lg">Sign Up</button>
+                    <button class="btn btn-outline btn-lg">Learn More</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!-- Footer -->
+    <footer class="footer p-10 bg-neutral text-neutral-content">
+        <nav>
+            <h6 class="footer-title">Services</h6>
+            <a class="link link-hover">Job Search</a>
+            <a class="link link-hover">Resume Builder</a>
+            <a class="link link-hover">Career Advice</a>
+            <a class="link link-hover">Salary Guide</a>
+        </nav>
+        <nav>
+            <h6 class="footer-title">Company</h6>
+            <a class="link link-hover">About us</a>
+            <a class="link link-hover">Contact</a>
+            <a class="link link-hover">Jobs</a>
+            <a class="link link-hover">Press kit</a>
+        </nav>
+        <nav>
+            <h6 class="footer-title">Legal</h6>
+            <a class="link link-hover">Terms of use</a>
+            <a class="link link-hover">Privacy policy</a>
+            <a class="link link-hover">Cookie policy</a>
+        </nav>
+    </footer>
+
+    <!-- Theme Toggle Script -->
+    <script>
+        // Theme toggle functionality
+        const themeToggle = document.querySelector('[data-toggle-theme]');
+        if (themeToggle) {
+            themeToggle.addEventListener('click', function () {
+                const currentTheme = document.documentElement.getAttribute('data-theme');
+                const newTheme = currentTheme === 'light' ? 'dark' : 'light';
+                document.documentElement.setAttribute('data-theme', newTheme);
+                localStorage.setItem('theme', newTheme);
+            });
+        }
+
+        // Load saved theme
+        const savedTheme = localStorage.getItem('theme') || 'light';
+        document.documentElement.setAttribute('data-theme', savedTheme);
+    </script>
```

Comment:
This should be abstracted to a js file.

Maybe have javascripts/index.js and then call that script here, means we don't have js in our html pages. 

Similarly to the css being in its own folder